docs/modules/data-structures/pages/listening-for-map-entries.adoc

```diff
@@ -99,21 +110,37 @@ C++::
 ----
 --
 
-C Sharp:: 
+C Sharp::
 +
--- 
+--
 [source,cs]
 ----
 
 ----
 --
 
-Node.js:: 
-+ 
--- 
+Node.js::
++
+--
 [source,javascript]
 ----
-
+const client = await Client.newHazelcastClient();
```

Comment:
One thing I noticed, Java provides whole working code sample. Can you also do the same? Example:

```js
const { Client, Predicates } = require('hazelcast-client');

async function main() {
    const client = await Client.newHazelcastClient();
    const map = await client.getMap('map');
    const listener = {
        added: (entryEvent) => {
            console.log('Entry added: ' + entryEvent.value);
        },
        removed: (entryEvent) => {
            console.log('Entry removed: ' + entryEvent.value);
        },
        updated: (entryEvent) => {
            console.log('Entry updated: ' + entryEvent.value);
        }
    };
    await map.addEntryListenerWithPredicate(listener, new Predicates.sql('surname = smith'), undefined, true);
    console.log('Entry listener registered');
    await client.shutdown();
}

main().catch(err => {
    console.error('Error occurred:', err);
    process.exit(1);
});

```